packages/block-library/src/post-featured-image/editor.scss

```diff
@@ -1,85 +1,92 @@
 // Provide special styling for the placeholder.
 // @todo: this particular minimal style of placeholder could be componentized further.
-.wp-block-post-featured-image.wp-block-post-featured-image {
-	// Style the placeholder.
-	.wp-block-post-featured-image__placeholder,
-	.components-placeholder {
-		justify-content: center;
-		align-items: center;
-		padding: 0;
-
-		// Hide the upload button, as it's also available in the media library.
-		.components-form-file-upload {
-			display: none;
-		}
+.wp-block-post-featured-image {
+	.block-editor-media-placeholder {
+		z-index: 1; // Need to put it above the overlay so the upload button works.
+		backdrop-filter: none; // Removes background blur so the overlay's actual color is visible.
```

Comment:
This creates accessibility as the upload button could blend in with the overlay/background. However, having the upload button blurred background causes the overlay to look very different than how it actually would look like.

Need suggestions/opinions on this.

Reply:
Have created separate issue https://github.com/WordPress/gutenberg/issues/44115 for tracking this.